app/scripts/controllers/citationInfo.ctrl.js

```diff
@@ -185,4 +185,23 @@ angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData
     }
   };
 
+  function legalRightsLink() {
```

Comment:
This ctrl.openLegalRightsLink and the legalRightsLink functions are basically duplicated betwen citationInfo.ctrl.js and courtSearchInfo.ctrl.js....It's enough that some common piece should be extracted. I see two options:

1) A new angular component for the button/link that invokes the function and the logic remains within that component's controller

2) A new angular factory (e.g. CourtRights.svc) which exposes an openRightsForCourt or something that takes the court as the param.